rpms/SPECS/3.9.0/wazuh-agent-3.9.0.spec

```diff
@@ -54,8 +54,14 @@ make clean
     make deps
     make -j%{_threads} TARGET=agent USE_SELINUX=yes PREFIX=%{_localstatedir}/ossec
 %else
+    %ifnarch x86_64
```

Comment:
This block can be simplified. `USE_MSGPACK_OPT` is set to `yes` by default (https://github.com/wazuh/wazuh/blob/6132b4e18e48454585711c0ce0e8b9391d534eb9/src/Makefile#L55). This means that you can remove the `%else` block and add only the `USE_MSGPACK_OPT=no` if you are building for i386.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/156/commits/df7c9c189c208465bb69c7bf0398a2e5f7a86bb7